publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/service/GenerateURLTemplate.java

```diff
@@ -29,7 +29,7 @@
     @Value("${generate.server.uri:{null}}")
```

Comment:
I guess generate.server.uri be default empty string as well if not set. Strange to have null as text in uri if not set. Better to have it as empty string.

Reply:
Even if we pass an empty value for generateServerPath like @Value("$(generate.server.path:{})"),
a null value is passed. So I have made changes in getUrl() method.

Reply:
Ok,,, I know I have implemented earlier with "@Value("${generate.server.uri:}")" and get an empty string if not set, don't remeber where in the EI code. But I guess its ok with your code change with null.